alpine/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM puppet/puppet-agent-alpine
+
+RUN apk add --update git \
+    && git clone https://github.com/jpogran/puppet-vscode.git
+
+ENTRYPOINT ["/usr/bin/ruby"]
+CMD ["/puppet-vscode/server/puppet-languageserver", "--no-stop", "--timeout=0"]
```

Comment:
timeout and nostop should only be disabled in debug scenarios.  Otherwise it creates longlived processes

Reply:
Should also expose the port number, perhaps as an ENV var?

Reply:
Ah, so we would want the container to fold up and go away on disconnect? My first pass here assumed you'd spin the container up and use it for multiple runs / connect disconnects. Pretty trivial to update.

Reply:
Probably best to expose the port number as an env, yeah.